search-service/src/main/resources/db/migration/V0_41__migrate_to_creator_right.sql

```diff
@@ -0,0 +1,45 @@
+WITH entities AS (
+    SELECT entity_id, count(*) as admin_right_count
+    FROM entity_access_rights
+    WHERE access_right = 'rCanAdmin'
+    GROUP BY entity_id
+)
+UPDATE entity_access_rights
+SET access_right = 'rIsOwner'
+WHERE entity_id IN (select entity_id from entities where admin_right_count = 1)
+AND access_right = 'rCanAdmin';
+
+-- set isOwner for entities with more than admin right
+WITH entities AS (
+    SELECT entity_id, count(*) as admin_right_count
+    FROM entity_access_rights
+    WHERE access_right = 'rCanAdmin'
+    GROUP BY entity_id
+), entities_more_than_one_admin AS (
+    SELECT entity_id
+    FROM entities
+    WHERE admin_right_count > 1
+), entities_with_oldest_date AS (
+    SELECT entity_id, min(created_at) as created_at
+    FROM temporal_entity_attribute
+    WHERE entity_id IN (select entity_id from entities_more_than_one_admin)
+    GROUP BY entity_id
+), entities_oldest_with_sub AS (
```

Comment:
`entities_with_oldest_sub` not `entities_oldest_with_sub`

Reply:
hmm, I am really looking for the oldest sub of each entity

Reply:
ah indeed, misread the comment